professor2.spec

```diff
@@ -1,15 +1,16 @@
-### RPM external professor2 2.3.3
+### RPM external professor2 2.4.2
 ## INITENV +PATH PYTHON3PATH %i/lib/python%{cms_python3_major_minor_version}/site-packages
+## INITENV +PATH PROF_VERSION %{realversion}
```

Comment:
@DickyChant , I think `PROF_VERSION` is not a `PATH` like variable. I suggest to change this to 
```
## INITENV SET PROF_VERSION %{realversion}
```

If this is needed in CMSSW env too then I would suggest to add the following in professor2.xml too
```
<runtime name="PROF_VERSION"  value="@PROF_VERSION@"/>
```
```

Reply:
You are indeed correct! I was using this to help somewhere for the Makefile to set the version. But I might actually be wrong about this since it might not be useful at all. I just forget to clean it up...

I will correct this with some test on whether we need it or not.